espnet/bin/asr_recog.py

```diff
@@ -257,7 +258,28 @@ def get_parser():
         default=0.999,
         help="Threshold probability for CTC output",
     )
-
+    # quantize model related
+    parser.add_argument(
+        "--quantize-config",
+        type=set,
+        default={torch.nn.Linear},
```

Comment:
I guess this does not work when non default value is set. It is OK to remove this and support only Linear at this moment.

Reply:
nn.Linear, nn.LSTM, nn.GRU, etc.